### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by QueenterJuma**
## What does this PR do?
This PR is raised to create a Portfolio using a Mobile template from Figma
##Description of the task to be completed
 [ x] Build a personal portfolio website.
 [x] Use a mobile Figma design.
 [x] Use images and backgrounds to enhance the look of the website
 [x] Use the [HTML&CSS linter] (https://github.com/microverseinc/linters-config/tree/master/html-css)
 [x] Validate the content of the email field to be in lowercase.
•	When the user submits the form, you check if the email is in lowercase.
•	If the validation is OK, the form is sent.
•	If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.
## How should it be tested
Clone the repository
```
git clone https://github.com/QueenterJuma/Portfolio_1/pulls 
```
Navigate to the folder
```
cd repo-name
```
Checkout the branch
```
git checkout branch_accecibility
```

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team! 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't contain a README file on the SOURCE of your repo. To make your repository looks professional, the first thing that the visitor should see is your README template on the main page of your repository. Please move your README file to the main source of your project so the visitors can see it on the main page of your repo

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -12,19 +12,16 @@ toggleMenu.addEventListener('click', () => {
...
     toggleMenu.classList.toggle('open');
     navigation.classList.toggle('open');
   });
 }
-
-/*form-validation*/
+/* form-validation */
 const form = document.getElementById('form');
-const email = form.elements[1].value
+const email = form.elements[1].value;
```
Comment (mckent05) [inline script.js:24]: - Your contact form isn't working as expected, according to the project requirements, If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. Your form doesn't submit even when the email is valid. A possible cause for this is in your `if` condition. Please fix this 👍 👍 

Your project:
ku,

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @QueenterJuma,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- All linter checks are passing 👍 👍 
- PR is professional 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @QueenterJuma 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**